public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -205,6 +205,7 @@ export function ServicesContent(props: ServicesProps) {
               />
             </EuiFlexItem>
           </EuiFlexGroup>
+          {props.filters.length > 0 && <EuiSpacer size="s" />}
```

Comment:
same here, Spacer should go to the field badge renderer. This don't have to repeat the code at multiple places.

Reply:
Moved into the filter.